src/c++/uds/userLinux/build/Makefile

```diff
@@ -15,7 +15,7 @@ vpath %.c $(SRC_UDS_DIR)
 # Flags for linking the shared library itself.
 SHLIBFLAGS = -lm -ldl -pthread -lrt $(shell getconf LFS_LDFLAGS)
 
-CFLAGS 	  = $(PLATFORM_CFLAGS) -I$(SRC_UDS_DIR)
+CFLAGS 	  = $(PLATFORM_CFLAGS) -DNDEBUG -I$(SRC_UDS_DIR)
```

Comment:
HM, if we're building with NDEBUG all the time now, maybe it makes sense to rename the token and switch the sense of ? Make the flag false be default in the code, and then let a dev set -DDEBUG if they want to use debug mode?  If you don't want to do that now, we can clean that up later.

Reply:
I think it would be better to clean this up later.